src/styl/site.styl

```diff
@@ -125,89 +128,90 @@ $margin-basic = ($gutter * 2)
 $margin-mid = ($gutter * 3.5)
 $margin-large = ($gutter * 5)
 
-$font-size-basic = 12px
-$font-size-extra = 48px
-$font-size-large = 36px
-$font-size-mid = 24px
-$font-size-small = 18px
-$font-size-tiny = 8px
+$font-size-basic = 11px
+$font-size-large = $font-size-basic * 1.5
+$font-size-small = $font-size-basic * .8
 
 grid-maker(96%, $gutter, s)
 
 space-maker($margin-micro, $margin-tiny, $margin-small, $margin-basic, $margin-mid, $margin-large)
 
-font-sizes($font-size-basic, $font-size-extra, $font-size-large, $font-size-mid, $font-size-small, $font-size-tiny)
+font-sizes($font-size-basic, false, $font-size-large, false, $font-size-small, false)
 
 @import 'responsive/mobile'
 
 @media screen and (min-width: 720px)
 
-	$gutter = 21px
+  $gutter = 21px
 
-	$margin-micro = ($gutter / 3)
-	$margin-tiny = ($gutter / 2)
-	$margin-small = ($gutter)
-	$margin-basic = ($gutter * 2)
-	$margin-mid = ($gutter * 3.5)
-	$margin-large = ($gutter * 5)
+  $margin-micro = ($gutter / 3)
+  $margin-tiny = ($gutter / 2)
+  $margin-small = ($gutter)
+  $margin-basic = ($gutter * 2)
+  $margin-mid = ($gutter * 3.5)
+  $margin-large = ($gutter * 5)
 
-	$font-size-basic = 14px
+  $font-size-basic = 17px
+  $font-size-large = $font-size-basic * 1.5
+  $font-size-small = $font-size-basic * .8
 
-	space-maker($margin-micro, $margin-tiny, $margin-small, $margin-basic, $margin-mid, $margin-large)
+  space-maker($margin-micro, $margin-tiny, $margin-small, $margin-basic, $margin-mid, $margin-large)
 
-	grid-maker(680px, $gutter, m)
+  grid-maker(96%, $gutter, m)
 
-	font-sizes($font-size-basic)
+  font-sizes($font-size-basic, false, $font-size-large, false, $font-size-small, false)
 
-	.mobile-only
-		display: none !important
+  .mobile-only
+  	display: none !important
```

Comment:
mixed tab/space indentation could be cause of those weird stylus compile issues

Reply:
yea it was

Reply:
idk how that happened in the first place tho